src/python/WMComponent/AnalyticsDataCollector/DataCollectAPI.py

```diff
@@ -76,13 +69,13 @@ def getJobSummaryByWorkflowAndSite(self):
             for x in results.get('rows', []):
                 data.setdefault(x['key'][0], {})
                 data[x['key'][0]].setdefault(x['key'][2], {})
-                #data[x['key'][0]][x['key'][1]].setdefault(x['key'][2], {})
+                # data[x['key'][0]][x['key'][1]].setdefault(x['key'][2], {})
```

Comment:
Oh, this was the code reformat (second commit). You shouldn't even see this change :-)